tasks/agent.py

```diff
@@ -614,6 +614,9 @@ def get_omnibus_env(
     # otherwise, ohai detect the number of CPU on the host and run the make jobs with all the CPU.
     if os.environ.get('KUBERNETES_CPU_REQUEST'):
         env['OMNIBUS_WORKERS_OVERRIDE'] = str(int(os.environ.get('KUBERNETES_CPU_REQUEST')) + 1)
+    # Forward the DEPLOY_AGENT variable so that we can use a higher compression level for deployed artifacts
+    if os.environ.get('DEPLOY_AGENT'):
+        env['DEPLOY_AGENT'] = os.environ.get('DEPLOY_AGENT')
```

Comment:
Are we using this new ENV value in this PR?

Reply:
Not in this PR directly. Basically we are spawning a new environment to execute omnibus in, and in omnibus we need a way to know if the artifact we're building is meant to be deployed or not, which is done through this environment variable.
If we do not forward it, we won't be able to know and will always use the default compression setting